_component_design/dropdowns.md

```diff
@@ -10,164 +10,47 @@ preview-image: components/preview-selectors.svg
-    </select>
-  </div>
-</div>
+### When to use
 
-## Disabled
-<div class="ui form">
-  <div class="field">
-    <label>Contact</label>
-    <select class="ui dropdown disabled" disabled>
-      <option value="">Make a selection</option>
-      <option value="Bart">Bart Renner</option>
-      <option value="Carmen">Carmen Broomes</option>
-      <option value="Elizabeth">Elizabeth Grotheus</option>
-      <option value="Eric">Eric Weidner</option>
-      <option value="Lane">Lane Fielder</option>
-      <option value="Mikey">Mikey Hougland</option>
-      <option value="Tommy">Tommy Shook</option>
-      <option value="Ty">Ty Taylor</option>
-    </select>
-  </div>
-</div>
+Drop-downs are used when a user is required to select a single option from
+a known list of options. For example, selecting a region on the
+Rackspace Cloud server create form will provide the following options:
 
-## Error
+* Chicago
+* Dallas
+* Hong Kong
+* London
+* Virginia
 
-<div class="ui form error">
-  <div class="field error">
-    <label>Contact</label>
-    <select class="ui dropdown">
-      <option value="">Make a selection</option>
-      <option value="Bart">Bart Renner</option>
-      <option value="Carmen">Carmen Broomes</option>
-      <option value="Elizabeth">Elizabeth Grotheus</option>
-      <option value="Eric">Eric Weidner</option>
-      <option value="Lane">Lane Fielder</option>
-      <option value="Mikey">Mikey Hougland</option>
-      <option value="Tommy">Tommy Shook</option>
-      <option value="Ty">Ty Taylor</option>
-    </select>
-    <div class="ui error message">
-      There is an error with the response to this question.
-    </div>
-  </div>
-</div>
+### Best practice
 
-## Multiple Select
+* Applying the disabled state to any options that are not currently available to
+  the user.
+* Keeping the drop-down pattern to a set of items that are easily parsable by a
+  user. While there is not a hard rule here for an upperbounds limit of items
+  to include a drop-down, once a drop-down data set exceeds 10 items it should
+  be looked at to determine if it could be streamlined.
+* Keeping the drop-down label viewable through all states of the dropdown. This
+  ensures users remember what label prompted the data set they are viewing.
 
-<div class="ui form">
-  <div class="field">
-    <label>Contact</label>
-    <select multiple="" class="ui dropdown">
-      <option value="">Make a selection</option>
-      <option value="Bart">Bart Renner</option>
-      <option value="Carmen">Carmen Broomes</option>
-      <option value="Elizabeth">Elizabeth Grotheus</option>
-      <option value="Eric">Eric Weidner</option>
-      <option value="Lane">Lane Fielder</option>
-      <option value="Mikey">Mikey Hougland</option>
-      <option value="Tommy">Tommy Shook</option>
-      <option value="Ty">Ty Taylor</option>
-    </select>
-  </div>
-</div>
+### Basic information
 
-## Type Ahead Select
+.. REQUIRES IMAGE
```

Comment:
@araiford - as per the PDF, this section here had an image that was not included on the PDF. Could you please add to this PR?